src/main/resources/events-mapping.json

```diff
@@ -1,114 +1,50 @@
 {
+  "dynamic": true,
```

Comment:
@RasonJ  is going to look at moving this down INTO the event_attributes stanza so you can have dynamic fields in there.   That would map to the specifiation that says "Additional Properties of any type allowed": https://o19s.github.io/ubi/docs/html/event.schema.html#event_attributes_object_additionalProperties